### PR TITLE
SER-86b2yvkc8/fix-business-health-score-card-image-flashing-issue

### PR DESCRIPTION
**Pull request by jwlim94**
This PR improves the visual consistency and user experience of the skeleton and blurred content.
1. Added Skeleton for Business Health Score
2. Skeleton Height Adjustment
3. Rounded Borders for Blurred Content

More detailed descriptions are self-commented below.

### CODE REVIEW

File: src/app/portal/dashboard/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -42,7 +44,7 @@ const Dashboard = ({
       <div className="flex flex-col lg:flex-row items-stretch gap-6 w-full">
         <div className="grow shrink basis-0">
           {isLoadingUser ? (
-            <div className="bg-gray-200 animate-pulse h-60 rounded-[20px]" />
+            <div className="bg-gray-200 animate-pulse h-full min-h-[300px] rounded-[20px]" />
```
Comment (jwlim94) [inline src/app/portal/dashboard/Dashboard/Dashboard.tsx:47]: This ensures the skeleton matches the card's height (the content displayed after loading).

File: src/app/portal/dashboard/Dashboard/LockedContent/LockedContent.tsx [hunk 2]
```diff
@@ -22,13 +23,20 @@ const LockedContent = ({
...
 
+  if (isLoading) {
+    return (
+      <div className="bg-gray-200 animate-pulse h-full min-h-[300px] rounded-[20px]" />
+    );
+  }
+
   return (
-    <div className="blur-container h-full min-h-[300px]">
+    <div className="blur-container h-full min-h-[300px] rounded-[20px]">
```
Comment (jwlim94) [inline src/app/portal/dashboard/Dashboard/LockedContent/LockedContent.tsx:39]: This ensures consistency between the locked and unlocked states of the component, as the blurred content (locked version) previously lacked the rounded borders that the unlocked version already had.

File: src/app/portal/dashboard/Dashboard/LockedContent/LockedContent.tsx [hunk 3]
```diff
@@ -22,13 +23,20 @@ const LockedContent = ({
...
   const onButtonClick = () => {
     onAction?.();
   };
 
+  if (isLoading) {
+    return (
+      <div className="bg-gray-200 animate-pulse h-full min-h-[300px] rounded-[20px]" />
+    );
+  }
+
```
Comment (jwlim94) [inline src/app/portal/dashboard/Dashboard/LockedContent/LockedContent.tsx:37]: This skeleton prevents the Business Health Score card image from flashing.

**Comment by landonwjohnson** [comment]
Probably a good idea to add a skeleton to the business score!! That's what I was thinking. (I wrote a comment about it earlier)

File: src/app/portal/dashboard/Dashboard/Dashboard.tsx (hunk 1 above)
Comment (landonwjohnson) [reply src/app/portal/dashboard/Dashboard/Dashboard.tsx:47]: nice catch!

File: src/app/portal/dashboard/Dashboard/LockedContent/LockedContent.tsx
Comment (landonwjohnson) [inline src/app/portal/dashboard/Dashboard/LockedContent/LockedContent.tsx:1]: I noticed this file has propTypes in it. which doesn't make a lot of sense if we are already using typescript. 
I do think we should get in the happen of writing default props, similar to what is happening to blurred being set to false by default.

**Review by landonwjohnson — APPROVED** [review]
looks good to me!

**Pull request merged**